src/legacy/ui/public/agg_types/metrics/lib/parent_pipeline_agg_controller.js

```diff
@@ -79,9 +70,6 @@ const parentPipelineAggController = function ($scope) {
 
     // we aren't creating a custom aggConfig
     if (metricAgg !== 'custom') {
-      if (!$scope.state.aggs.find(agg => agg.id === metricAgg)) {
```

Comment:
This logic was moved to `MetricAggParamEditor` component: https://github.com/elastic/kibana/pull/37979/files?file-filters%5B%5D=.tsx&owned-by%5B%5D=#diff-d09c7d350e3017c52e2e151fba97ef27R54